src/charm.py

```diff
@@ -65,13 +65,15 @@ def _on_install(self, _: ops.InstallEvent) -> None:
                 headers={"Authorization": default_token},
                 data={"password": self.state.api_password},
                 timeout=10,
+                verify=False,
```

Comment:
Is this a certificate issue?

Reply:
The certificate might be self signed and it won't be issued for `localhost`